portugues/colaborando.md

```diff
@@ -13,6 +13,8 @@ Você:
 
 [Cria um aplicativo melhor](cria/melhor.md)
 
+[Procura um amigo geek e reinventa um app bem melhor!](reinventar-app.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.